src/compiler/commandLineParser.ts

```diff
@@ -1,6 +1,55 @@
 namespace ts {
     /* @internal */
     export const compileOnSaveCommandLineOption: CommandLineOption = { name: "compileOnSave", type: "boolean" };
+
+    // NOTE: The order here is important to default lib ordering

```

Comment:
i would expand on this comment and say that the order of the list here is the same order the lib files have in the generated program, and add a note to see use in `createProgram`